sound/soc/sof/ipc4-loader.c

```diff
@@ -14,9 +14,6 @@
 #include "sof-priv.h"
 #include "ops.h"
 
-/* The module ID includes the id of the library it is part of at offset 12 */
```

Comment:
@aiChaoSONG I am not comfortable with the terminology "loadable module". Any module can be be a loadable module with IPC4. What we really want to say is that "Any module loaded from an external library will have a non-zero library ID...."